pdk/1.3/modules/ROOT/pages/policies-pdk-configure-features-contracts.adoc

```diff
@@ -0,0 +1,300 @@
+= Using Contracts Validation Library functions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+NOTE: To view an example policy project that uses Flex Gateway Policy Development Kit (PDK)'s Contracts Validation library, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/{template-policies-url-ver-var}/contracts-validation/README.md[Contracts Validation Policy Example^].
+
+xref:api-manager::api-contracts-landing-page.adoc[Contracts] between a API instances and applications give the application access to the instance based on SLA tiers. Client applications can validate their contract by providing credentials that consist of the client ID and client secret. The PDK Contracts Validation library provides functions to validate client credentials and implement a custom client ID enforcement policy.
```

Comment:
I think "between a API instances" should be spelled as "between API instances". Also "Client applications can validate their contract" should be "Client applications can validate their contracts".